include/sys/epoll.h

```diff
@@ -47,7 +47,9 @@ struct epoll_event {
 	uint32_t events;
 	epoll_data_t data;
 }
-#ifdef __x86_64__
```

Comment:
Only `__x86_64__` is packed, the others must not be packed. See also: <https://git.musl-libc.org/cgit/musl/tree/include/sys/epoll.h>

They did this to ensure compatibility when running 32 bit x86 and 64 bit x64 code on the same machine. But for all other cases they didn't do this. So instead of modifying `sys/epoll.h` we should modify the test.